docs/getting-started-guides/photon-controller.md

```diff
@@ -209,7 +209,7 @@ backend nodes
 
 #### Option 2: Ingress Controller
 
-Using an [ingress controller](docs/user-guide/ingress) may also be an
+Using an [ingress controller](/concepts/services-networking/ingress/#ingress-controllers) may also be an
```

Comment:
Looks like you're missing "/docs" the beginning of the URL:

`/docs`/concepts/services-networking/ingress/#ingress-controllers